front/app/api/areas/useAreas.test.ts

```diff
@@ -0,0 +1,58 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import useAreas from './useAreas';
+
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+
+import createQueryClientWrapper from 'utils/testUtils/queryClientWrapper';
+import { areasData } from './__mocks__/useAreas';
+import { IAreasQueryParams } from './types';
+
+const apiPath = '*/:type/:id/areas';
+
+const params: IAreasQueryParams = {
+  pageNumber: 1,
+  pageSize: 5000,
+};
```

Comment:
These are the fallback values, so strictly speaking not necessary to pass?

Reply:
True, it's not strictly necessary, just making it more explicit.